CONTRIBUTING.md

```diff
@@ -27,7 +27,8 @@ So you want to contribute your changes back to WPILib. Great! We have a few cont
 - Bug reports and fixes
     - We will generally accept bug fixes without too much question. If they are only implemented for one language, we will implement them for any other necessary languages. Bug reports are also welcome, please submit them to our GitHub issue tracker.
 - While we do welcome improvements to the API, there are a few important rules to consider:
-    - Features must be added to both WPILibC and WPILibJ, with rare exceptions.
+    - Features must be added to Java (WPILibJ), C++ (WPILibC), with rare exceptions.
+        - Most of Python (RobotPy) is created by wrapping WPILibC in python. However, new features to the command framework should also be submitted to [robotpy-commands-v2](https://github.com/robotpy/robotpy-commands-v2) as the command framework is reimplemented in Python.
```

Comment:
I would say "is created by wrapping WPILibC with pybind11 via robotpy-build", since "wrapping in python" implies that the work to be done involves writing python... which it doesn't.